frontend/src/components/App.tsx

```diff
@@ -258,23 +245,25 @@ export default class App extends Component {
         if (r) return r;
 
         return (
-            <ErrorBoundary>
-                {/* {IsDev && <DebugDisplay />} */}
-                <Switch>
-                    {/* Login (and callbacks) */}
-                    <Route exact path="/login" component={Login} />
-                    <Route path="/login/callbacks/:provider" render={p => <LoginCompletePage provider={p.match.params.provider} match={p.match} />}></Route>
-
-                    {/* Default View */}
-                    <Route path="*">
-                        <Layout style={{ height: '100vh', background: 'transparent', overflow: 'hidden' }}>
-                            {isEmbedded() ? null : <AppSide />}
-                            <AppContent />
-                        </Layout>
-                    </Route>
-                </Switch>
-                <FeatureErrorCheck />
-            </ErrorBoundary>
+            <ChakraProvider theme={redpandaTheme}>
```

Comment:
@sago2k8 @rikimaru0345 When the app is embedded it doesn't control this.  We need to supply the provider from cloud-ui when we embed.